rules/falco_rules.yaml

```
@@ -1902,6 +1902,32 @@
   priority: INFO
   tags: [container, cis, mitre_privilege_escalation, mitre_lateral_movement]
 
+# These capabilities were used in the past to escape from containers
+- macro: excessively_capable_container
+  condition: >
+    thread.cap_permitted contains CAP_SYS_ADMIN
+    or thread.cap_permitted contains CAP_SYS_MODULE
+    or thread.cap_permitted contains CAP_SYS_RAWIO
+    or thread.cap_permitted contains CAP_SYS_CHROOT
+    or thread.cap_permitted contains CAP_SYS_PTRACE
+    or thread.cap_permitted contains CAP_SYS_BOOT
+    or thread.cap_permitted contains CAP_SYSLOG
+    or thread.cap_permitted contains CAP_DAC_READ_SEARCH
+    or thread.cap_permitted contains CAP_NET_ADMIN
+    or thread.cap_permitted contains CAP_NET_RAW
+
+- rule: Launch Excessively Capable Container
+  desc: Detect container started with a powerful set of capabilities. Exceptions are made for known trusted images. 
+  condition: >
+    container_started and container
+    and not excessively_capable_container
```

```suggestion
    and excessively_capable_container
```

It may be late in the day but I believe you didn't want this negation.

Feel free to pick my suggestion if I'm not getting this wrong!

---

Of course, you are right! Thanks!

---

Please remember to add "Co-Author-By" when someone helps you somehow :)

---

Done 🙂